spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/RabbitProperties.java

```diff
@@ -467,6 +483,11 @@ public void setSize(Integer size) {
 	public static class Listener {
 
 		/**
+		 * Container type.
+		 */
+		private ContainerType type = ContainerType.SIMPLE;
```

Comment:
Thanks @garyrussell - We've created a dedicated issue to figure this out. Can you please move your comment over there? #9173